zulipterminal/core.py

```diff
@@ -236,6 +237,12 @@ def show_about(self) -> None:
             'area:help'
         )
 
+    def show_user_info(self, user_id: int) -> None:
+        self.show_pop_up(
+            UserInfoView(self, user_id, 'User Information (up/down scrolls)'),
+            'area:help'
```

Comment:
Let's definitely add a new area for this - if nothing else, this is confusing to read.

Reply:
Sure. I'll propose `area:info` in my coming iteration

Reply:
We already have areas for more specific names, so I'd suggest `area:user` may be more appropriate?